content/cloud-docs/workspaces/state.mdx

```diff
@@ -41,6 +41,12 @@ Terraform's built-in [`terraform_remote_state` data source](/language/state/remo
 
 Terraform Cloud automatically manages API credentials for `terraform_remote_state` access during [runs managed by Terraform Cloud](/cloud-docs/run#remote-operations). This means you do not usually need to include an API token in a `terraform_remote_state` data source's configuration.
 
+## Upgrading State
+
+A workspace's state version can be upgraded to a new Terraform version using no-operation (empty) applies. Empty applies are a [type of run mode](/cloud-docs/run/modes-and-options) that allows Terraform Cloud to proceed with an apply when a plan produces no changes.   
```

Comment:
Hmm, this is great but might benefit from a bit of expansion. 

I think the key points to try and fit into here somehow are: 

- Sometimes the state format changes between Terraform versions.
- Terraform will automatically perform any needed state upgrades when you apply a run. 
- If you choose a version of Terraform that isn't compatible with your existing state, it will fail the run and tell you which version you need to apply with first. You can solve the problem by switching to that version and running a full apply.
- You can use a no-op (empty) apply to perform these state upgrades without needing to make any config changes.